package.json

```diff
@@ -48,17 +48,20 @@
         "postcustom-element-json": "lerna exec --ignore \"{@spectrum-web-components/{base,bundle,clear-button,close-button,iconset,modal,shared,opacity-checkerboard,styles,custom-vars-viewer,reactive-controllers,vrt-compare,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- node ../../tasks/check-cem.js",
         "postdocs:analyze": "node ./scripts/add-custom-properties.js --src=\"projects/documentation/custom-elements.json\"",
         "postinstall": "patch-package && yarn get-ready",
-        "postlerna-publish": "yarn build:react && lerna publish from-package --message \"chore: release new Reach Wrapper versions #publish\"",
+        "postlerna-publish": "yarn publish:react",
+        "postpublish:react": "git reset --soft HEAD^ && git reset && git prune",
```

Comment:
Just `git reset` will default to HEAD only. Is this really needed since the HEAD Is already at that commit after doing `git reset --soft HEAD^`

Reply:
This is definitely at the very edge of my git knowledge here, but the goal I'm trying to achieve is to `git reset --soft HEAD^` to remove the latest commit, and then `git reset` to remove all of the changes that are released back into the branch from that commit. Maybe there's a single command version of that?

Reply:
May be something like this:https://git-scm.com/docs/git-reset#Documentation/git-reset.txt---mixed